guide/sequelize/currency.md

```diff
@@ -277,7 +277,7 @@ const storedBalances = await Users.findAll();
 storedBalances.forEach(b => currency.set(b.user_id, b));
 ```
 
-In our ready event, we want to sync our currency collection with the database for easy access later.
+We want to sync our currency collection with the database for easy access later in our ready event.
```

Comment:
I think I prefer the former. This can interpreted as "do this so that you can have easy access to it later _inside_ your ready event", instead of just "inside your ready event, do this so that you can have easy access to it later"